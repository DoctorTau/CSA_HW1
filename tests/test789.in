451
-9335 6931 4458 -7929 -1441 6034 -5831 -2433 1109 -1475 -2394 8166 -8889 -1036 -8777 4544 3434 -9289 4974 -4982 6128 7249 -290 1176 717 -8323 -7711 -5470 -6088 -5569 1022 -9470 5899 -2730 1771 -9015 -1072 -8165 -7338 3910 -6504 4274 -5921 -938 3240 1518 7391 7513 -1631 -8608 9695 216 -4143 3697 9891 -8857 6644 -9667 1568 6614 9290 -581 554 2506 -5437 4269 5897 -318 6435 -664 -4221 2537 5148 349 -7359 775 -432 -9143 -7203 8876 8574 1163 4117 3573 4476 8401 -8681 2126 5215 -4333 -3157 6094 9720 6979 -7162 4161 8643 107 -5644 8655 5798 3685 -4226 -8350 5203 6040 -2019 -6248 -6246 -5440 1134 -1851 -3775 8364 5623 2857 -2946 -7752 -4978 -734 1786 2984 -2314 2552 -9233 1325 -2928 -7528 -862 -5155 5632 -5091 9045 -3032 6693 -4529 -6651 144 -9176 894 -278 -6084 -1836 8797 -3423 2464 8077 4798 3738 -514 -8197 -3767 -4474 9824 -3151 -6509 6304 6897 5075 -4716 -7478 -9876 1238 -1984 -8186 5972 9497 -6078 -4955 -1599 -3992 -4627 8662 2225 -8520 4023 2593 8228 -5049 -5694 9826 -218 -8139 396 4666 -4094 -1537 4881 -3480 5630 -5483 -4354 7957 815 -840 -5177 3214 1542 -4088 -1261 2382 4335 7719 -6530 -6191 828 788 -3229 -6916 -8938 -1364 2356 8002 3874 -1524 -6897 5922 9240 -7585 5632 8617 -929 -8469 6369 -3207 -1811 9112 4039 4748 8804 -4447 -2861 -1923 -3568 8677 5032 4758 4461 -429 -8648 -2733 206 -9050 -5702 -4025 -5301 712 -4106 1884 -6643 774 -7472 -3983 -9930 -6946 1809 1456 9962 1597 9387 2356 6199 -5526 5092 -6926 -9568 1658 1836 -1666 6244 -9372 9135 8216 -2880 -4239 7881 -6678 -791 -5695 -281 937 6557 2378 -1395 -8541 -4841 8865 -960 8644 -6006 8120 4626 7755 -8375 -5158 -4900 893 3448 -2489 -5534 6574 7376 3038 -8222 4317 -1285 589 5383 -4466 -6043 -5600 -3350 -8540 7946 -7273 -2501 -1770 606 5973 7913 2628 3270 -6193 8394 -1530 5992 1879 -2380 -7567 -1380 2780 -885 4096 6941 4476 7981 6767 -9827 -9340 -4926 -8128 1924 327 -3187 8232 -8758 -5404 -3686 -893 6248 3141 4023 1402 7734 3099 9715 2430 2263 7877 -7126 4873 -6790 2069 7400 -3456 -9481 -6820 -6234 3784 -8452 6871 398 301 1337 2158 -4170 -6009 -302 -4787 6158 -9525 -4765 6734 -6503 -2130 4322 6694 3231 -4753 7187 -5597 9001 1155 4054 -7799 -4580 4712 -8455 -472 -7018 -7560 -4305 -6005 -2480 -6097 3915 8790 -8202 -9871 -1969 -7621 -4221 -9602 6782 -9813 -8221 2231 2688 -8082 5345 -96 1470 3752 2162 -1569 -1105 9491 9475 6851 5109 7874 2259 7627 1125 3149 2810 3520 5904 7157 9905 -8444 -7663 678 -9394 -1387 9012 -224 9320 -9371 -4187 -7495 