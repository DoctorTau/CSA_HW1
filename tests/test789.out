-2394 -1036 -4982 -290 -5470 -6088 -9470 -2730 -1072 -7338 -6504 -938 -8608 -318 -664 -432 -7162 -5644 -4226 -8350 -6248 -6246 -5440 -2946 -7752 -4978 -734 -2314 -2928 -7528 -862 -3032 -9176 -278 -6084 -1836 -514 -4474 -4716 -7478 -9876 -1984 -8186 -6078 -3992 -8520 -5694 -218 -4094 -3480 -4354 -840 -4088 -6530 -6916 -8938 -1364 -1524 -3568 -8648 -9050 -5702 -4106 -7472 -9930 -6946 -5526 -6926 -9568 -1666 -9372 -2880 -6678 -960 -6006 -5158 -4900 -5534 -8222 -4466 -5600 -3350 -8540 -1770 -1530 -2380 -1380 -9340 -4926 -8128 -8758 -5404 -3686 -7126 -6790 -3456 -6820 -6234 -8452 -4170 -302 -2130 -4580 -472 -7018 -7560 -2480 -8202 -9602 -8082 -96 -8444 -9394 -224 